server/models/Availability.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+
+const availabilitySchema = new mongoose.Schema({
+    userId: {
+        type: mongoose.Schema.Types.ObjectId,
+        required: true,
+        ref: 'User',
+    },
+    petSitterId: {
+        type: mongoose.Schema.Types.ObjectId,
+        required: true,
+        ref: "User",
+    },
+    activeOn: {
+        type: String,
+        default: "1 Event Type",
+    },
+    timeZone: {
+        type: String, 
+        default: 'Pacific Time - US & Canada',
+    },
+    workingHours: [{
```

Comment:
This design won't fit the use case of the project, as `wokringHours` can contain duplicates with the same day/start/end times